### DISCUSSION TITLE
how to load intel ethernet driver on talos

### DISCUSSION THREAD

**Discussion by yingwana3** [siderolabs/talos#7525] [Q&A]
Hi, after installing talos 1.4.6 (using iso) on baremetal platform, we find that only mellanox drivers are there
ls /lib/modules/6.1.37-talos/kernel/drivers/net/ethernet
mellanox

We want to know how to load intel ethernet drivers on talos. For example, 
ls /lib/modules/6.1.37-talos/kernel/drivers/net/ethernet
mellanox
intel

Thanks

**Comment by smira** [comment]
Which driver you're talking about specifically? Many of them are compiled in.

**Reply by yingwana3** [reply]
You mean they are built in kernel instead of built as kernel module?

**Reply by yingwana3** [reply]
I have chekced the config file and found all Intel ethernet drivers have been built with y option. So they have already been built in kernel.
